### PR TITLE
doc: update links in links.txt to point to TechDocs

### PR DESCRIPTION
**Pull request by richabp**
Updated links of NCS older version bundles
and known issues to point to TechDocs.

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -225,53 +225,53 @@
 
 .. ### Source: developer.nordicsemi.com
```
Comment (divipillai) [inline doc/nrf/links.txt:226]: Shouldn't all the "docs.nordicsemi.com" url be moved to ".. ### Source: docs.nordicsemi.com" ? Most of them are still uder developer.nordicsemi.com.

Comment (umapraseeda) [reply doc/nrf/links.txt:226]: May be the whole section be renamed as .. ### Source: docs.nordicsemi.com, all the developer URLs could be moved out to a separate section under ### Source: developer.nordicsemi.com

Comment (divipillai) [reply doc/nrf/links.txt:226]: We already have a section with ### Source: docs.nordicsemi.com. We could move these and other urls in this section to "### Source: docs.nordicsemi.com"

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -319,6 +267,179 @@
...
+.. _`nRF Connect SDK v2.1.0 webinar`: https://www.nordicsemi.com/Events/2022/Webinar-Exciting-new-features-in-nRF-Connect-SDK-v210?utm_campaign=2022%20Webinars&utm_source=Documentation&utm_medium=Link%20to%20webinar%20%7C%20from%20documentation&utm_content=Link%20to%20webinar%20%7C%20from%20documentation
+.. _`nRF Connect SDK v2.0.0 webinar`: https://www.nordicsemi.com/Events/2022/Webinar-Exciting-new-features-in-nRF-Connect-SDK-v2?utm_campaign=2022%20Webinars&utm_source=developer.nordicsemi.com&utm_medium=Webinar%20registration%20%7C%20nRF%20Connect%20SDK%20v2.0&utm_content=Webinar%20registration%20%7C%20nRF%20Connect%20SDK%20v2.0
+.. _`nRF Connect SDK v1.9.0 webinar`: https://www.nordicsemi.com/Events/2022/Webinar-Exciting-new-features-in-nRF-Connect-SDK-v19?utm_campaign=2022%20Webinars&utm_source=developer.nordicsemi.com&utm_medium=In-line%20text%20%7C%20nRF%20Connect%20SDK%20v1.9%20%7C%20Link%20to%20webinar
+.. _Adding Device Firmware Update (DFU/FOTA) Support in nRF Connect SDK: https://www.nordicsemi.com/Events/2024/Webinar-Adding-Device-Firmware-Update-DFU-FOTA-Support-in-nRF-Connect-SDK
```
Comment (umapraseeda) [inline doc/nrf/links.txt:356]:
```suggestion
.. _`Adding Device Firmware Update (DFU/FOTA) Support in nRF Connect SDK`: https://www.nordicsemi.com/Events/2024/Webinar-Adding-Device-Firmware-Update-DFU-FOTA-Support-in-nRF-Connect-SDK
```

**Review by divipillai — APPROVED** [review]

**Pull request merged**